### DISCUSSION TITLE
How can I save the data in the form when I render sections in this form?

### DISCUSSION THREAD

**Discussion by REDLIGHTFILMS** [react-hook-form/react-hook-form#1949] [Q&A] [answered]
How can I save the data in the form when I render sections in this form? Like in the example "nested Forms". So I want to click through the sections with a stepper from Material UI and output everything at the end. How can I do this theoretically?

**Comment by keiya01** [comment, marked as answer]
You can use `useFormContext` with `FormContext`.

https://react-hook-form.com/api/#useFormContext

**Reply by REDLIGHTFILMS** [reply]
thx :)
